Docker/start-container.sh

```diff
@@ -6,11 +6,11 @@ PUID=${PUID:-`id -u squeezeboxserver`}
 PGID=${PGID:-`id -g squeezeboxserver`}
 
 usermod -o -u "$PUID" squeezeboxserver
-groupmod -o -g "$PGID" squeezeboxserver
+groupmod -o -g "$PGID" nogroup
 
 umask 0002
 
 #Add permissions
-chown -R squeezeboxserver:users /config /playlist
+chown -R squeezeboxserver:nogroup /config /playlist /lms
 
-/usr/bin/perl /lms/slimserver.pl --user squeezeboxserver --prefsdir /config/prefs --logdir /config/logs --cachedir /config/cache
+su squeezeboxserver -c '/usr/bin/perl /lms/slimserver.pl --prefsdir /config/prefs --logdir /config/logs --cachedir /config/cache'
```

Comment:
I think using `sudo` instead of `su` would be better.
```
exec sudo -u squeezeboxserver -E /usr/bin/perl /lms/slimserver.pl --user squeezeboxserver --prefsdir /config/prefs --logdir /config/logs --cachedir /config/cache
```
`sudo` then also needs to be added to the `apt-get install` in the Dockerfile

Reply:
If you don't have any commands after it does not make a difference. Adding sudo is just overhead in that case.

But if you have other commands following that you want to execute them as root than you should use sudo as with su you are changing user also for next commands.